src/main/java/com/pp/demo/UserControler.java

```diff
@@ -0,0 +1,34 @@
+package com.pp.demo;
+
+import org.springframework.stereotype.*;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class UserControler {
+
+    private UserRepository userRepository;
+
+    public UserControler(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @ResponseBody
+    @GetMapping("/users")
+    public String usersList() {
+
+        return userRepository.printUserList();
+    }
+
+    @RequestMapping("/add")
+    public String addUser(@RequestParam String imie, @RequestParam String nazwisko, @RequestParam Integer wiek) {
```

Comment:
Zrob tak by wporwadzenie `/add?nazwisko?abc&wiek=12` też przekierowywało na stronę z błędem

Reply:
I zmień nazwy parametrów na angielskie :P Jesli chcesz przyjmowac w url dalej polskie nazwy to skorzystaj z parametru `name` w adnotacji @RequestParam